.github/workflows/docs.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.13
+      - name: Update package index
+        run: sudo apt-get update
```

Comment:
@john-science this update was needed for this to pass CI. I am not sure if we will run into issues with its removal from other workflows, but I just added what was needed to get this PR to pass CI

Reply:
@opotowsky 

Humph. This adds 3-5 minutes to CI.

I am confused... how did you come across this? Because I saw the docs build on the PR that removed this, and then another couple PRs after that.

Reply:
docs builds began to fail, reliably. I re-added it (failure message said to run `update`) and they began passing.

Reply:
super bummer it adds 5 min though. I don't know of a way around it :-(